site2/docs/reference-pulsar-admin.md

```diff
@@ -2422,28 +2422,28 @@ Usage
 $ pulsar-admin topics remove-delayed-delivery tenant/namespace/topic
 ```
 
-### `get-max-producers`
+### `get-maxProducers`
```

Comment:
The correct form is :`get-max-producers`

Reply:
@freeznet Can you double confirm the changes?

Reply:
@sijia-w just as jennifer mentioned, there are some code errors should be fixed first. such as `get-max-producers` should be added to topic commands.